data_download.md

```diff
@@ -23,4 +25,20 @@ The data portal offers a number of services:
   
 We encourage you to use the web services in preference to the ftp service. It will become increasingly difficult to access ftp servers in the future – support for ftp in web browsers has already been largely withdrawn. INTERMAGNET is likely to withdraw the ftp service in the future.
 
+## <a name='downloading_data_using_dois'></a> Downloading Data Using DOIs
+
+From 2015 onwards the entire definitive data set from Intermagnet is updated and published annually. 
+This data set is called the Intermagnet Reference Data Set (IRDS).
+
+- IRDS 2020: [https://doi.org/10.5880/INTERMAGNET.1991.2020](https://doi.org/10.5880/INTERMAGNET.1991.2020)
+- IRDS 2019: [https://doi.org/10.5880/INTERMAGNET.1991.2019](https://doi.org/10.5880/INTERMAGNET.1991.2019)
+- IRDS 2018: [https://doi.org/10.5880/INTERMAGNET.1991.2018](https://doi.org/10.5880/INTERMAGNET.1991.2018)
+- IRDS 2017: [https://doi.org/10.5880/INTERMAGNET.1991.2017](https://doi.org/10.5880/INTERMAGNET.1991.2017)
+- IRDS 2016: [https://doi.org/10.5880/INTERMAGNET.1991.2016](https://doi.org/10.5880/INTERMAGNET.1991.2016)
+- IRDS 2015: [https://doi.org/10.5880/INTERMAGNET.1991.2015](https://doi.org/10.5880/INTERMAGNET.1991.2015)
+- 2014 publication: [https://doi.org/10.5880/INTERMAGNET.2014](https://doi.org/10.5880/INTERMAGNET.2014)
+- 2013 publication: [https://doi.org/10.5880/INTERMAGNET.2013](https://doi.org/10.5880/INTERMAGNET.2013)
+
+To use the data you will need the BGS software : [Imcdview download](https://geomag.bgs.ac.uk/data_service/intermagnet/home.html) / [installer](https://geomag.bgs.ac.uk/data_service/intermagnet/imcdviewInstaller/install.htm) 
```

Comment:
We aren't updating the installer any more (and will probably remove it soon). There's a new Java installer for all BGS Intermagnet software - the place to look is under the heading "Intermagnet Software Installer" on the page https://geomag.bgs.ac.uk/data_service/intermagnet/home.html - I could add an anchor to this heading so you can link directly to it if that helps?